### ISSUE TITLE
Auto mode templates resolver behavior changed

### ISSUE THREAD

**Issue by stanislas-m** [gobuffalo/buffalo#1000]
Since commit https://github.com/gobuffalo/buffalo/commit/b7ceff565f5a9a6d001cd3eba6984f52afbccba3, the templates resolver behavior changed.

I use a model called `RoomProviders`:

```go
roomProviders := models.RoomProviders{}
// ...
return c.Render(200, r.Auto(c, roomProviders))
```

Before https://github.com/gobuffalo/buffalo/commit/b7ceff565f5a9a6d001cd3eba6984f52afbccba3, the template was resolved to `templates/room_providers/index.html`. Now it's resolved to `templates/roomproviders/index.html` (without the underscore).

**Comment by markbates** [comment]
@stanislas-m congrats you've opened the 1000th issue! ;)

@paganotoni you broke it, you bought it. :)

**Comment by paganotoni** [comment]
closed by #1002
